src/internals/strings/Strings.ts

```diff
@@ -15,6 +15,7 @@ export namespace Strings {
 
   /**
    * Get the length of a string.
+   * @warning - 🔥🔥🔥does not work with emojis since they are multiple characters🔥🔥🔥
```

Comment:
This is kind of an edge case. Probably worth mentioning in the docs, but maybe not in inline comment (which show up on hover in editors running the TS language server)